action.yml

```diff
@@ -25,6 +25,10 @@ inputs:
     description: 'Tag of the built image'
     required: false
     default: ''
+  archs:
+    description: 'architecture to build on'
```

Comment:
Please use the description format as other inputs have. Mention, please, if multiple archs can be used, and how they should be specified. 
Update please also readme with information about this new input.

Reply:
ok